lib/vacations/vacation.ex

```diff
@@ -0,0 +1,45 @@
+defmodule GlassFactoryApi.Vacations.Vacation do
+  @moduledoc """
+  Defines the Vacations of a member
+  """
+
+  alias GlassFactoryApi.Vacations.Vacation
+
+  @type t() :: %Vacation{
+          id: String.t(),
+          upcoming: boolean(),
+          daily_hours: String.t(),
+          start_date: String.t(),
+          end_date: String.t(),
+          days: String.t(),
```

Comment:
String or integer?

Reply:
Yes, this should be an integer, but the Glass Factory API returns a string, and we decided to keep the same type. at least in this first version 😄